model/ftn/ww3_ounf.ftn

```diff
@@ -1161,10 +1161,7 @@
 !
             ! Atmospheric momentum
             ELSE IF ( IFI .EQ. 1 .AND. IFJ .EQ. 8 ) THEN
-              !! Note - TAUA and TAUADIR read in from .ww3 file are TAUAX,TAUAY
-!/RTD              ! Rotate x,y vector back to standard pole
```

Comment:
Did these lines get deleted by accident? Or was it intentional?

Reply:
yes, sorry about that. Definitely not intentional. That should be fine now and ww3_ounf.ftn file shouldn't be in the changes.